src/main/java/game/GameFlow.java

```diff
@@ -8,25 +8,32 @@
 public class GameFlow {
 
     private final Ui ui;
-    private final HumanPlayer humanPlayer;
+    private HumanPlayer humanPlayer;
     private final Computer computer;
 
-    public GameFlow(Ui ui, HumanPlayer humanPlayer, Computer computer) {
+    public GameFlow(Ui ui, Computer computer) {
         this.ui = ui;
-        ui.welcomePlayer();
-        this.humanPlayer = humanPlayer;
         this.computer = computer;
+        gameSetUp();
     }
 
     public void run() {
-        Move userMove = humanPlayer.makeMove();
+        Move userMove = humanPlayer.makeMove(ui);
+
+        Move computerMove = computer.makeMove(ui);
 
-        Move computerMove = computer.makeMove();
-        ui.declareComputerMove(computerMove.getName());
+        printResult(userMove.playAgainst(computerMove));
```

Comment:
I think you can inline `userMove` and `computerMove`. No need to save it in a variable first.